### PR TITLE
dvc 0.70.0

### PR DESCRIPTION
**Pull request by efiop**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/dvc.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@ class Dvc < Formula
 
   desc "Git for data science projects"
   homepage "https://dvc.org"
-  url "https://github.com/iterative/dvc/archive/0.68.1.tar.gz"
-  sha256 "a072ebf2151213c61ac7e580e51dc1cecefa4cd840e4f7ae1927d6710312cfe0"
+  url "https://github.com/iterative/dvc/archive/0.70.0.tar.gz"
+  sha256 "e8632310dc2eaab9b182d25fbc26e0a321ee1beb8cd7693ebb3ee53946e2c9d7"
   revision 2
```
Comment (Bo98) [inline Formula/dvc.rb]: You should remove this line. A new version means that we start from revision 0 again.

**Review by Bo98 — CHANGES_REQUESTED** [review]

File: Formula/dvc.rb (hunk 1 above)
Comment (efiop) [reply Formula/dvc.rb]: Oops, sorry, missed that one. Thanks!

Comment (efiop) [reply Formula/dvc.rb]: @Bo98 Removed the `revision` line.

Comment (Bo98) [reply Formula/dvc.rb]: Thanks!

**Review by Bo98 — APPROVED** [review]

**Pull request closed without merging**